webextensions/api/extension.json

```diff
@@ -82,6 +82,11 @@
               "safari_ios": {
                 "version_added": "15"
               }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": false,
+              "deprecated": true
```

Comment:
getBackgroundPage is not deprecated

Reply:
Sorry, must've misunderstood your comment, reverted